alcove/mcp_server.py

```
@@ -0,0 +1,277 @@
+    collections = [collection] if collection else None
+    raw = query_text(query, n_results=n_results, collections=collections)
+    documents = raw.get("documents", [[]])[0]
+    metadatas = raw.get("metadatas", [[]])[0]
+    distances = raw.get("distances", [[]])[0]
+
+    source_ids = set(source_ids_include or [])
+    source_group_ids = set(source_group_ids_include or [])
+    language = (language_filter or "").strip()
+
+    results = []
+    for doc, meta, dist in zip(documents, metadatas, distances):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)